config/passport.js

```diff
@@ -117,13 +127,14 @@ module.exports = (passport) => {
           }
           if (!user) {
             console.log(profile);
+
             user = new User({
               name: profile.displayName,
               email: (profile.emails && profile.emails[0].value) || '',
               username: profile.username,
               provider: 'facebook',
               facebook: profile._json,
-              avatar: profile._json.picture || profile.json.picture.data.url
+              picture: profile._json.picture || profile.json.picture.data.url || profile.photos[0].value
```

Comment:
This will break some other part of the code as they rely on avatar instead of the picture object. That's why the avatar was retained instead. same meaning different names.